docs/proposals/prop-1-proposals.md

```diff
@@ -0,0 +1,153 @@
+good in principle sometimes turn out to be impractical when subjected to the test of implementation. Often it 
+makes sense for the prototype implementation to be made available as a separate PR to the repo (making sure to 
+appropriately mark the PR as a WIP). Another option is to include a prototype as an auxiliary file along with the 
+proposal. 
+
+## Review and Resolution
+
+Proposals are discussed on the anonlink mailing list. The possible paths of the status of a proposal are:
+
+![](./prop-1/resolution-process.png)
+
+Eventually, after discussion, there may be a consensus that the proposal should be accepted – see the next section 
+for details. At this point the status becomes `Accepted`.
+
+Once a Proposal has been `Accepted`, the implementation or change must be completed. When the reference implementation
+is complete and incorporated into the main source code repository, the status will be changed to `Final`.
+
+To allow gathering of additional design and interface feedback before committing to long term stability for a 
+feature or standard library API, a Proposal may also be marked as "Provisional". This is short for "Provisionally
+Accepted", and indicates that the proposal has been accepted for inclusion, but additional user feedback is needed
+before the full design can be considered `Final`. Unlike regular accepted proposals, provisionally accepted
+proposals may still be `Rejected` or `Withdrawn` even after the related changes have been included in a release.
+
+Wherever possible, it is preferable to reduce the scope of a proposal to avoid the need to rely on the `Provisional`
+status (e.g. by deferring some features to later).
+
+A Proposal can also be assigned status `Deferred`. The author or any core developer can assign this status
+when no progress is being made on a Proposal.
+
+A Proposal can also be `Rejected`. Perhaps after all is said and done it was not a good idea. It is still important to
+have a record of this fact. The `Withdrawn` `status is similar—it means that the author themselves has decided that
+the Proposal is actually a bad idea, or has accepted that a competing proposal is a better alternative.
+
+When a Proposal is `Accepted`, `Rejected`, or `Withdrawn`, the status should be updated accordingly. In addition to
+updating the status field, the Resolution header should be added with a link to the relevant PR or thread in the mailing
+list archives.
+
+Proposals can also be superseded by a different Proposal, rendering the original obsolete. The status should be updated
+to `Replaced` and the `Replaced-By` and `Replaces` headers should be added to the original and new Proposals 
+respectively.
+
+Process proposals may have a status of `Active` if they are never meant to be completed, e.g. this proposal.
+
+## Technical Specifications
+
+### Document
+
+Proposal documents live in the `docs/proposals` folder of the `anonlink` library. Proposals are written
```

Comment:
Each proposal only has one status - history is available via git

Reply:
On the first question - I was thinking it might be best in an `anonlink-docs` or `anonlink-project` repo but wanted to get feedback from everyone else